src/domain/app/header/Header.tsx

```diff
@@ -161,81 +168,86 @@ const Header: React.FC = () => {
 
   return (
     <ClassNames>
-      {({ css, cx }) => (
-        <Navigation
-          id={PAGE_HEADER_ID}
-          menuOpen={menuOpen}
-          onMenuToggle={toggleMenu}
-          menuToggleAriaLabel={t('navigation.menuToggleAriaLabel')}
-          skipTo={`#${MAIN_CONTENT_ID}`}
-          skipToContentLabel={t('navigation.skipToContentLabel')}
-          className={cx(styles.navigation, css(theme.navigation), {
+      {({ cx }) => (
+        <HDSHeader
+          className={cx(styles.navigation, {
             [styles.hideNavRow]: noNavRow,
           })}
-          onTitleClick={goToHomePage}
-          title={t('appName')}
-          titleUrl={`/${locale}${ROUTES.HOME}`}
-          logoLanguage={
-            locale === 'sv' ? /* istanbul ignore next */ 'sv' : 'fi'
-          }
+          id={PAGE_HEADER_ID}
+          theme={theme.navigation}
+          languages={languageOptions}
+          onDidChangeLanguage={changeLanguage}
         >
-          <Navigation.Row>
+          <HDSHeader.SkipLink
+            skipTo={`#${MAIN_CONTENT_ID}`}
+            label={t('navigation.skipToContentLabel')}
+          ></HDSHeader.SkipLink>
+          <HDSHeader.ActionBar
+            title={t('appName')}
+            titleHref={`/${locale}${ROUTES.HOME}`}
+            onTitleClick={goToHomePage}
+            frontPageLabel={t('common.home')}
+            logoHref={`/${locale}${ROUTES.HOME}`}
+            logo={
+              <Logo
+                src={locale === 'sv' ? logoSvDark : logoFiDark}
+                size="medium"
+                alt={t('navigation.logo')}
+              />
+            }
+            menuButtonAriaLabel={t('navigation.menuToggleAriaLabel')}
+          >
+            <HDSHeader.LanguageSelector
+              ariaLabel={t(`navigation.languages.${locale}`)}
+            />
+            {!isMobile && (
+              <HDSHeader.ActionBarItem
+                icon={<IconSearch />}
+                label={t('navigation.tabs.searchEvents')}
+                id="action-bar-search-events"
+                onClick={handleSearch}
+              />
+            )}
+            {Boolean(authenticated && user) ? (
```

Comment:
Boolean is redundant. This can be simplified to 
```suggestion
            {authenticated && user ? (
```

Reply:
https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/258/commits/1c3592072e018a96138e9ab2fd98b431ac048fff